scripts/requirements.in

```diff
@@ -1,9 +1,10 @@
 pypdf
+aiohttp
 azure-identity
 azure-search-documents==11.4.0b11
 azure-ai-formrecognizer
 azure-storage-blob
 azure-storage-file-datalake
-openai[datalib]==0.28.1
+openai[datalib]>=1.3.5,<2.0.0
```

Comment:
You can remove upper here too

Reply:
Also you can now remove the ignore from dependabot of openai package, as I was only ignoring it while we weren't on 1.0